values.schema.json

```diff
@@ -7,7 +7,9 @@
     "DataSourceEntry": {
       "type": "object", "title": "DataSourceEntry", "additionalProperties": false,
       "properties": {
-        "url": {"type": "string", "title": "url source to query for data"}
+        "url": {"type": "string", "title": "url source to query for data"},
+        "topics": {"type": "array", "title": "topics the data applies to", "default": ["policy_data"], "items": { "type": "string" }},
```

Comment:
Did you check that the default actually works properly when isn't set ?

Reply:
I'll check :)

Reply:
@RazcoDev It doesn't work :)
But I don't really need it - so just got rid of it